openshift/tests-extension/pkg/helpers/commands.go

```diff
@@ -0,0 +1,33 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+)
+
+// findK8sClient returns "oc" if available, otherwise "kubectl".
+func findK8sClient() (string, error) {
+	clients := []string{"oc", "kubectl"}
+	for _, c := range clients {
+		if err := exec.Command(c, "version", "--client").Run(); err == nil {
+			return c, nil
+		}
+	}
+	return "", fmt.Errorf("no Kubernetes CLI client found (tried oc, kubectl)")
+}
```

Comment:
Something like this basically exists...
I think using `exec.LookPath` is probably a better approach.
https://github.com/openshift/operator-framework-operator-controller/blob/2f8f54b35eb3cc626f9693bf23c0b475bdbdca54/openshift/tests-extension/test/olmv1-incompatible.go#L389-L394

Reply:
Although a slice/for-range mechanism could be used.

Reply:
Sure,  I explicitly tried not to do the refactoring here, thinking of doing a follow-up to make it easier to get the review. But yep, you are right. It is all done now. Thank you a lot 👍